CMakeLists.txt

```diff
@@ -109,7 +109,9 @@ endif()
 
 find_package(Filesystem REQUIRED)
 
-if(NOT WIN32)
+if(WIN32)
+  find_package(krabsetw REQUIRED)
```

Comment:
Could you also add a line for copying the license file down at line 220?

Reply:
Ah yes, added, thanks!